File: cpp/program/ProdukElektronik.cpp
#include <iostream>
#include <string>

class ProdukElektronik {
protected:
    std::string idProduk;
    std::string namaProduk;
    std::string merk;

public:
    // Constructor
    ProdukElektronik(const std::string& idProduk, const std::string& namaProduk, const std::string& merk)
        : idProduk(idProduk), namaProduk(namaProduk), merk(merk) {}

    // Getters
    std::string getIdProduk() const { return idProduk; }
    std::string getNamaProduk() const { return namaProduk; }
    std::string getMerk() const { return merk; }

    // Setters
    void setIdProduk(const std::string& idProduk) { this->idProduk = idProduk; }
    void setNamaProduk(const std::string& namaProduk) { this->namaProduk = namaProduk; }
    void setMerk(const std::string& merk) { this->merk = merk; }
};

File: java/program/ProdukElektronik.java
public class ProdukElektronik {
    private String idProduk;
    private String namaProduk;
    private String merk;

    public ProdukElektronik(String idProduk, String namaProduk, String merk) {
        this.idProduk = idProduk;
        this.namaProduk = namaProduk;
        this.merk = merk;
    }

    // Getters
    public String getIdProduk() { return idProduk; }
    public String getNamaProduk() { return namaProduk; }
    public String getMerk() { return merk; }

    // Setters
    public void setIdProduk(String idProduk) { this.idProduk = idProduk; }
    public void setNamaProduk(String namaProduk) { this.namaProduk = namaProduk; }
    public void setMerk(String merk) { this.merk = merk; }
}